README.md

```diff
@@ -556,6 +560,10 @@ These are items relevant *only* on OS X.
 
 - You can perform and script most Windows system administration tasks from the command line by learning and using `wmic`.
 
+- Another option to get Unix look and feel under Windows is [Cash](https://github.com/dthree/cash). Note that only very few Unix commands and command-line options are available in this environment.
+
+- An alternative option to get GNU developer tools (such as GCC) on Windows is [MinGW](http://www.mingw.org/) and its [MSYS](http://www.mingw.org/wiki/msys) package, which provides utilities such as bash, gawk, make and grep. MSYS doesn't have all the features compared to Cygwin.
```

Comment:
I suggest to add: MinGW is particularly useful for creating native Windows ports of Unix tools.